### ISSUE TITLE
Nested tilde selectors fail to retain tilde

### ISSUE THREAD

**Issue by matharden** [Kronuz/pyScss#351]
This code…
```sass
div ~ {
  p,
  blockquote {
    background-color: blue;
  }
}
```

should output…
```sass
div ~ p,
div ~ blockquote {
  background-color: blue;
}
```

but incorrectly discards the tildes and returns…
```sass
div p,
div blockquote {
  background-color: blue;
}
```

My current work-around is to write it like this…
```sass
div {
  ~ p,
  ~ blockquote {
    background-color: blue;
  }
}
```

**Comment by matharden** [comment]
This is also a problem with child (`>`) selectors. The following code fails to retain the `>`
```sass
div > {
  a,
  strong {
    background-color: blue;
  }
}
```

**Comment by eevee** [comment]
Fixed in 1.3.5.
